tests/Functional/Execution/ExecutionTest.php

```diff
@@ -60,12 +62,12 @@ public function testNoSchemaIsProvided()
     {
         $this->expectException(\TypeError::class);
 
-        /** @var ExecutionResult $executionResult */
-        Execution::execute(null, parse('{field}'));
+        graphql(null, parse('{field}'));
```

Comment:
Looks good! 👍

Reply:
It's suppose not to work, the second parameter should be string . which is weird....

Reply:
Shouldn't this be `graphql(null, '{field}')`?